Node.md

```diff
@@ -19,25 +19,25 @@ You can use either Yarn or NPM to install the dependencies. This document will u
 
 ### Polyfills
 
-Using the SDK with Node also requires the use of some polyfills. In your application's entrypoint (`main.ts`, `index.ts`, or something similar), you will need to register those polyfills:
+Using the SDK with Node.js also requires the use of some polyfills. In your application's entrypoint (`main.ts`, `index.ts`, or something similar), you will need to register those polyfills:
 
 ```ts
 // main.ts
 
-import wrtc = require('node-datachannel/polyfill');
+const wrtc = require('node-datachannel/polyfill');
```

Comment:
Node.js supports two module formats: CommonJS (`require`) and ESM (`import`). While they can be mixed together in some programs, the expected syntax for each format is different. If we wanted to use ESM import statements it would look a little like reversed Python:

```ts
import wrtc from 'node-datachannel/polyfill';
```

However after updating the example code to use ESM, it appears the Viam TypeScript SDK does not have an ESM compatible module export for Node.js at the moment. So I've updated the example code to use CommonJS for now.

Reply:
This file works fine for me in Node 20 mod your API key secret fix

Reply:
`import wrtc = require('node-datachannel/polyfill')` works in Node 20? That's quite odd since it doesn't work in the REPL:

<img width="851" alt="image" src="https://github.com/user-attachments/assets/7f74f54d-fb06-4a4d-a096-44e1ed453bd6">

Could `tsc` be transpiling the code to be valid?

Reply:
Yes tsc or tsx would transpile to js since node can't run typescript.

Reply:
I believe eslint/the community in general prefers ESM style imports in TS? At least i often see lint errors when i do `const foo = require('bar')`

Reply:
Node 22 can run TypeScript with the experimental type stripping flag and would fail with the example code as is. I think we should have valid JS in the examples, even if tsc papers over the syntax errors.

Reply:
> I believe eslint/the community in general prefers ESM style imports in TS? At least i often see lint errors when i do const foo = require('bar')

Yes, ESM is the official standard and should be used when possible. However, the current build output of the TS SDK cannot be imported as ESM in Node.js:

<img width="886" alt="image" src="https://github.com/user-attachments/assets/c168d4d9-79d2-4242-a1a1-35e39e5db5a7">

(The above example in the REPL is using the dynamic ESM import syntax but the same error applies when running `main.ts` with static ESM imports)